wordpress-mobile/WordPress-iOS
# History / Revisions: Detail with Attributed strings

Refs. #10304 
Branched from #10468 

This PR uses a stack view with 2 labels (Title and content) to display the revision data with no render HTML. It also adds some functions to combine the revision diff operations in order to display additions and deletions.

![simulator screen shot - iphone x - 2018-11-17 at 17 31 35](https://user-images.githubusercontent.com/912252/48676007-63dedf00-eb58-11e8-96eb-497ddfad694b.png)

What's change from the #10468 PR:
- `RevisionDiffViewController` doesn't implement the Aztec textView anymore. Instead there's a _stackview_ with 2 labels, all wrapped into a _scrollview_.

- `RevisionDiff` implements an extension with 2 computed properties to return the title and content attributed string.

- `DiffAbstractValue+Attributes` is a new extensions which contains the functions used to combine and return the final attributed string. 

**To test:**
• Edit a post with revision history
• Tap … then History
• Tap a revision
• Ensure the UI behaves as expected
